### PR TITLE
Oguntunjiharlimatodunola patch 2

### LINKED ISSUE #45: feat: Add a contributing guide
### Type of feature

🍕 Feature

### Current behavior

There's no contributing guide.

### Suggested solution

Since we've had some great contributions on this course and excitement around continued contributions, it's important to have a contributing guide specific to this repo. I'm thinking specifically about issue #43, we could have better supported our contributor by ensuring they knew more.

We link to the main contributing guide from the docs, but we should add specifics for this repo:

1. how to add a new section, including conventions for numbering, 
2. the use of markdown, 
3. the need to add new sections to the sidebar.
4.  how to run the project locally. I suggest two methods: using the docsify CLI or using VS Code Go Live feature.
5.  how to add translations

So I would suggest adding a contributing.md with a link to https://docs.opensauced.pizza/contributing/introduction-to-contributing/ followed by the specifics

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Lymah123**
## Description

**This pull request makes the following changes:**  
- closes #45 

Type of feature
🍕 Feature

Current behavior
There's no contributing guide.

Suggested solution
Since we've had some great contributions on this course and excitement around continued contributions, it's important to have a contributing guide specific to this repo. I'm thinking specifically about issue https://github.com/open-sauced/intro/issues/43, we could have better supported our contributor by ensuring they knew more.

We link to the main contributing guide from the docs, but we should add specifics for this repo:

how to add a new section, including conventions for numbering,
the use of markdown,
the need to add new sections to the sidebar.
how to run the project locally. I suggest two methods: using the docsify CLI or using VS Code Go Live feature.
how to add translations
So I would suggest adding a contributing.md with a link to https://docs.opensauced.pizza/contributing/introduction-to-contributing/ followed by the specifics

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Oguntunjiharlimatodunola** [comment]
@BekahHW , this is the new pull request I created for the contributing guide. I already linked the issue number in the repo. Let me know if this is exactly what you asked me to do.

**Comment by Lymah123** [comment]
@CBID2 , this is the PR.

**Comment by CBID2** [comment]
> @CBID2 , this is the PR.

@Lymah123, these are the commits that you did for PR #44. I highly recommend closing this PR and waiting until that one is merged.

**Review by BekahHW — CHANGES_REQUESTED** [review]
The only file that should be in this PR is the contributing guide. 

Make sure you follow the conventions of file naming. 

Instead of `contributing guide.md` it should be `CONTRIBUTING.md`

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md:14]: This isn't the current setup for this project, so let's make the updates to this section. You can keep steps 1 and 2. Here's what should follow.

There are two ways to run this project locally:
1. Use the Go Live feature in VS Code.
2. Install the docsify CLI. 

### Docsify CLI
1. In your terminal, install the CLI, `npm i docsify-cli -g`
2. Once the CLI is installed, run `docsify serve`

Once you've done this, you should be able to navigate to the docs site locally.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md:16]: You can remove lines 16-29

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md:34]: Make this a heading:

### A Step-by-Step Guide to adding a new section

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md:41]: This is great. Make a note here to follow the naming conventions. It could be something like this:

> Note: Be sure to follow naming conventions. Notice that files are not capitalized and in place of spaces between words there are hyphens.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: Instead of line 45, let's add this:

In this special course repository, numbers represent chapters. Consider where the new section would logically fall in the course. If you're not sure, please comment on the issue and tag a maintainer, asking for help. If the section you're adding isn't a chapter, you can omit the number.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: Section three already talks about this, so let's omit this.

File: contributing guide.md
Comment (Lymah123) [reply contributing guide.md]: Okay, noted.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: I can't think of when a new section wouldn't require being added to the sidebar. Adding the file to the sidebar is necessary for the user to discover the content.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: Let's make this a heading too:

### Markdown Basics

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: Let's move this section to where sidebar is mentioned earlier. Steps:

1. Navigate to `_layouts/sidebar.js`
2. Add New Section link: 
	- To add a new section link, insert a new line with the following format:
	- [Intro](/01-intro.md) -- be sure to capitalize the section title.
	- Replace "Intro" with the title of your new section.
	- Replace "/01-intro.md" with the relative path to the Markdown file of your new section.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: I don't think we need docs here.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: I think the live server is native to VS Code right now.  So I don't think this needs installed, but I'll double check.

File: contributing guide.md
Comment (BekahHW) [reply contributing guide.md]: Ok, I see this is below. So there are two options here, you can mention it here and link to that section, or you can move the directions up.

File: contributing guide.md
Comment (BekahHW) [inline contributing guide.md]: @CBID2 could you review the adding translations section since you just worked on it?

**Review by BekahHW — CHANGES_REQUESTED** [review]
Hey @Lymah123! Thanks for putting together this comprehensive guide! This is going to go a long way to support new contributors. I've added some comments in the PR for some updates that need made. Let me know if you have any questions.

File: contributing guide.md
Comment (CBID2) [reply contributing guide.md]: Sure @BekahHW

File: CONTRIBUTING.md
Comment (CBID2) [inline CONTRIBUTING.md:232]: [proofreading] It helps mentioning specific tools so that developers who want to translate documentation can see how it's done.
```suggestion
- Consider using translation tools or services like [Crowdin](https://crowdin.com/) and [Google Translate](https://translate.google.com/about/) to assist with the translation process.
```

File: CONTRIBUTING.md
Comment (CBID2) [inline CONTRIBUTING.md:231]: [suggestion] When I worked on the French translation of the Intro course with Geoffrey, we would split the work into parts, so consider mentioning the importance of collaboration.   
```suggestion
- Maintain consistency in terminology and style throughout the translated documentation.
- Work with another contributor who speaks the language you're translating too.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Lymah123. I've made some suggestions in the section about best translation practices.  Keep it up! :)

**Comment by Lymah123** [comment]
Thank you @BekahHW and @CBID2 .

File: contributing guide.md
Comment (Lymah123) [reply contributing guide.md]: @BekahHW , I think you are right but let me know when you double check.

**Review by geoffreylgv — COMMENTED** [review]
Hi, great work @Lymah123 

what if we refer to [The style guide](https://github.com/open-sauced/pizza-verse/blob/main/style-guide.md) and [Markdown website](https://www.markdownguide.org/basic-syntax/) as well

**Comment by Lymah123** [comment]
Hi, @geoffreylgv. Thank you!

Great, I will include the links in the guide. Thanks for the suggestion.

**Comment by CBID2** [comment]
@Lymah123, any updates on this?

**Comment by Lymah123** [comment]
> @Lymah123, any updates on this?

I have not gotten feedback from @BekahHW .

**Comment by BekahHW** [comment]
@Lymah123 did you add @geoffreylgv's suggestions? 

@CBID2 does the translation section look good on your end?

**Comment by Lymah123** [comment]
> @Lymah123 did you add @geoffreylgv's suggestions?
> 
> @CBID2 does the translation section look good on your end?

Yes @BekahHW,I have added @geoffreylgv suggestions and @CBID2 has given her suggestions as well. I have worked on them all. If you can sill check again to be very sure all the necessary changes have been made.

**Review by BekahHW — APPROVED** [review]
LGTM

**Comment by geoffreylgv** [comment]
> > @Lymah123 did you add @geoffreylgv's suggestions?
> > @CBID2 does the translation section look good on your end?
> 
> Yes @BekahHW,I have added @geoffreylgv suggestions and @CBID2 has given her suggestions as well. I have worked on them all. If you can sill check again to be very sure all the necessary changes have been made.

Congratulations you all @Lymah123, @CBID2, @BekahHW  🥳

**Comment by Lymah123** [comment]
> LGTM

I am glad! 

Thanks to y'all  @BekahHW , @CBID2 and @geoffreylgv for your unwavering support.

**Pull request merged**
